t/ui/03-source.t

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env perl
```

Comment:
Don't add this line. Please have a look at our documentation how to start unit tests. If you're using my openQA-helper you can also simply use `openqa-test t/ui/03-source.t`.

Reply:
Ok, it seems like other tests have the `#!/usr/bin/env perl` comment as well. Does anybody else in the team know what it is used for? Tests should usually be invoked with `prove`, right? (Or via the `Makefile` which also uses `prove`.)

Reply:
Yes, we rely on this for syntax highlighting in some editors